src/main/resources/static/api.yaml

```diff
@@ -1385,6 +1396,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/recovery/verifyotp:
```

Comment:
Can you think of better name? /patients/profile/recovery-confirm

Reply:
will rename it